inc/components/class-component.php

```diff
@@ -557,6 +557,17 @@ private function hydrate_config(): self {
 			$new_config = call_user_func_array( $this->config_callback, [ $new_config ] );
 		}
 
+		/**
+		 * Filter the config values after the config callback fires.
+		 *
+		 * Note: This filter uses `after_hydration` instead of `post_hydration`
+		 * to avoid confusion with `post` in the WordPress context.
+		 *
+		 * @param array  $config Config for this component.
+		 * @param string $name   Name of this component.
+		 */
+		$new_config = apply_filters( 'wp_irving_component_config_after_hydration', $new_config, $this->get_name() );
```

Comment:
Can we just shorten this one to `wp_irving_component_config` or is there a reason to retain that filter name for some other purpose? The shorter version would have parity with the `wp_irving_component_children` filter which fires after the `children_callback` method and child hydration is finished.

Reply:
I think it's useful to clarify since the other one is `wp_irving_component_config_pre_hydration`?

Reply:
Then should we rename the `wp_irving_component_children` filter in the same way?